### ISSUE TITLE
Mekanism fusion reactor sometimes stops working

### ISSUE THREAD

**Issue by Libright1558** [NillerMedDild/Enigmatica2Expert#1282]
**Modpack version**
1.65d

**Can the issue be reproduced?**
Yes

**Logs**
N/A

**Issue**
This issue happened when i went into the compact machine and went back to the overworld(maybe going to the other dimensions triggers this issue ) 
I am sure the fusion reactor is inside the chunk,and the chunk is loaded.Fuel is enough to keep the reactor running.(In these pictures,the reactor had been reignited by me)

![2019-07-22_09 55 59](https://user-images.githubusercontent.com/19789411/61601312-71a8af00-ac67-11e9-9469-544cf5c5a09c.png)
![2019-07-22_09 56 18](https://user-images.githubusercontent.com/19789411/61601313-71a8af00-ac67-11e9-9abd-f4ed4a165877.png)
![2019-07-22_09 56 31](https://user-images.githubusercontent.com/19789411/61601314-71a8af00-ac67-11e9-925b-952e5973d20e.png)
![2019-07-22_09 56 43](https://user-images.githubusercontent.com/19789411/61601315-72414580-ac67-11e9-8cb5-225eb21fc08b.png)
![2019-07-22_09 56 57](https://user-images.githubusercontent.com/19789411/61601318-72414580-ac67-11e9-8160-fecd276a4678.png)

**Comment by NillerMedDild** [comment]
You will need to provide a lot more information for this to get anywhere to be honest - Pictures of the setup would be good for starters.

**Comment by Libright1558** [comment]
[mekanism/Mekanism#4665](https://github.com/mekanism/Mekanism/issues/4665)
I don't know whether the issue is related to above.

**Comment by NillerMedDild** [comment]
Thanks for the screenshots. What are you using to load the chunk?

**Comment by Libright1558** [comment]
FTB Utilities

**Comment by NillerMedDild** [comment]
Please try with the IC2 chunkloader.

**Comment by Libright1558** [comment]
Just test it out,this issue didn't be triggered,but i can't confirm this issue won't happen again.
![2019-07-23_19 03 21](https://user-images.githubusercontent.com/19789411/61709822-a2313b80-ad82-11e9-82e0-bce9b58951bf.png)
![2019-07-23_19 04 10](https://user-images.githubusercontent.com/19789411/61709823-a2c9d200-ad82-11e9-9eec-d96c5ddde355.png)

**Comment by NillerMedDild** [comment]
Cool, feel free to reopen the issue if it happens again.

**Comment by Libright1558** [comment]
This issue isn't solved......
![2019-07-24_23 35 28](https://user-images.githubusercontent.com/19789411/61808456-0250dc00-ae6e-11e9-8bb6-2f6dab651978.png)

**Comment by NillerMedDild** [comment]
Your fuel generation setup is chunkloaded right?

**Comment by Libright1558** [comment]
> Your fuel generation setup is chunkloaded right?

Yes, the chunk where the Loader is placed is loaded,i set fuel generation in that chunk.

**Comment by Frikkle** [comment]
Random question/side thought while scrolling by this issue again: do you produce enough fuel? When I had the exact same setup (i.e. one TEP for Brine and one TEP for Lithium, I did not have fast enough Brine/Lithium production to have a sustainable fuel supply; my reactor ran dry after a while.

Do your Thermal Evaporation Plants stay completely full while the reactor is running? Or are they (slowly) draining? If so, they just run dry and as a result the reactor shuts off. At that point the fuel production continues again until it's all filled up; you then log in seeing lots of fuels and no running reactor, hence confusion.

Let me know :)

**Comment by Libright1558** [comment]
I hope these pictures can help you find out the problems.
![2019-07-30_18 35 23](https://user-images.githubusercontent.com/19789411/62128127-5c3a2180-b306-11e9-972f-6c5b60201c27.png)
![2019-07-30_18 35 30](https://user-images.githubusercontent.com/19789411/62128128-5c3a2180-b306-11e9-970d-69f1e7e3517c.png)
![2019-07-30_18 36 23](https://user-images.githubusercontent.com/19789411/62128130-5cd2b800-b306-11e9-905a-64b6bca4fc69.png)
![2019-07-30_18 36 36](https://user-images.githubusercontent.com/19789411/62128132-5cd2b800-b306-11e9-8d56-89377fc3625f.png)
![2019-07-30_18 37 42](https://user-images.githubusercontent.com/19789411/62128133-5cd2b800-b306-11e9-98c0-f21553e37104.png)
![2019-07-30_19 05 45](https://user-images.githubusercontent.com/19789411/62128134-5d6b4e80-b306-11e9-9a3c-a4351ccbf0ff.png)
![2019-07-30_19 05 51](https://user-images.githubusercontent.com/19789411/62128136-5d6b4e80-b306-11e9-909b-f52d0ba8b3d4.png)

**Comment by Frikkle** [comment]
Could you answer the question/give feedback to the theory I posted? More pictures don't help me that much as they are static and do not tell me anything about how your setup behaves over time.

**Comment by Frikkle** [comment]
Additionally, could you provide us some more information about:
- whether this issue persists after updating to 1.66c
- whether you are using stock E2E (as in, with no additional mods and/or config changes)

**Comment by Libright1558** [comment]
The amount of fuel didnt change after playing for 27 minutes. (00:00~01:44
;27:44~28:36)
[https://youtu.be/oFdS72dtHBo](url)

**Comment by Libright1558** [comment]
> Additionally, could you provide us some more information about:
> 
> * whether this issue persists after updating to 1.66c





> Additionally, could you provide us some more information about:
> 
> * whether this issue persists after updating to 1.66c 
Unknown.
> * whether you are using stock E2E (as in, with no additional mods and/or config changes)
Yes,i don't add/remove any mod or change config.

**Comment by Frikkle** [comment]
Unknown because you have not seen it happen in 1.66c/1.67? Or because you did not update?

**Comment by Frikkle** [comment]
This issue seems to be reported to Mekanism already and is still open there: https://github.com/mekanism/Mekanism/issues/5246

**Comment by Frikkle** [comment]
The issue with Mekanism is traced and placed on the backlog of the Devs to be fixed in the near future.

**Comment by Libright1558** [comment]
> Unknown because you have not seen it happen in 1.66c/1.67? Or because you did not update?

I didnt see it happened in 1.66c.Anyway,thank you for your help.

**Comment by Frikkle** [comment]
I'll keep this issue open and monitor its progress with the Mekanism devs. When a release/solution is provided I'll update and close this.

**Comment by KrakenG7** [comment]
May I add, I was having the same issues by pumping DT fuel directly into the reactor, but after injecting deuterium and tritium individually in addition to DT fuel directly, and setting the burning rate to the minimum, the reactor stopped randomly shutting off on me

**Comment by EveOfFrost** [comment]
I think I found the issue
Maybe, as I'm playing on a Stoneblock 3 modpack and this is happening to me as well.

I think the issue is lag spiking, as whenever a large enough lag spike occurs it causes mekanism multiblocks to disconnect and reconnect. I found this out when placing large quantities of mechanical pipes as placing and or breaking them would cause all the connected pipes to disconnect from each other and reconnect. And this ended up causing my fusion reactor and thermoelectric boilers to disconnect and reconnect as well.

**Comment by NielsPilgaard** [comment]
Interesting, nice observation. I'm afraid there's not much we can do in that regard.

**Comment by EveOfFrost** [comment]
Fair
Will say, the DT fuel thing still works to keep it active, probably cus of how stupidly high it causes the temperature to sky rocket to even on 2 injection rate.

I don't know what version the pack uses, but this is still persistent on Stoneblock 3 version 1.4.2 and 1.5.0

**Comment by KEJWII** [comment]
I actually have the same problem too
- Mekanism-1.12.2-9.8.3.390
- MekanismGenerators-1.12.2-9.8.3.390
Fusion reactor shuts down when I exit the void dimension (UtilityWorlds-1.12.2_1.1.0)
im use (chunkloaders-1.2.3-forge-mc1.12) for chunk loaders to load these parts of the dimension

**Comment by BenLeclerc** [comment]
I think I was having a similar problem to this in Stoneblock 4, with a max Fission Reactor in a 45x45x45 compact machine.  Was sometimes completely reforming and voiding all the water and fissile fuel I had in there and setting the config back to .1mb/tick.

The solution that seems to be working is instead of relying only on the FTBTeams chunk loading, I added a Dimensional Stabilizer (also from Mekanism) to load the 9 chunks in that compact machine.  I figured that the way that it was loading chunks would be more favorable to the reactor not breaking, and it seems to have worked.  Could be helpful for anyone else who runs into this problem.  

My theory on this is perhaps that the other chunk loaders are sometimes doing 1 chunk at a time (or at least not all chunks at the same time), and that the largest reactor is in at least 4 chunks as a result, and loading only one of them, even for a single tick, would break the machine.  The Dimensional Stabilizer has a config to load up to a 5x5 area of chunks, so the chunks are guaranteed to all be loaded at the same time.
